proposals/NNNN-Clarification-of-SDL-App-Icon-display-sequence-and-display-order.md

```diff
@@ -0,0 +1,78 @@
+
+ 1. Set the priority of the SDL app according to the AppHMIType priority in Table 1.
+ 2. When there are multiple AppHMITypes set in one SDL App, the AppHMIType with the highest priority is used, as the priority of that SDL App.
+ 3. If multiple SDL apps have the same AppHMIType set, sort them according to the order the SDL apps registered in the policy table.
+
+<b>Table 1.</b> AppHMIType priority list
+
+| Priority | AppHMIType |
+|:-:|:-:|
+| 1 | NAVIGATION |
+| 2 | MEDIA |
+| 3 | MESSAGE |
+| 4 | INFORMATION |
+| 5 | COMMUNICATION |
+| 6 | SOCIAL |
+| 7 | PROJECTION |
+| 8 | REMOTE_CONTROL |
+| 9 | SYSTEM |
+| 10 | DEFAULT |
+| 11 | BACKGROUND_PROCESS |
+| 12 | TESTING |
+
+At first, The priority information of AppHMIType in Table 1 is stored in the policy table, the SDL Core and default policy table (sdl_preloaded_pt.json). Then, the first searched information is used as the AppHMIType priority list (priority list). During the first launch, there is no policy table, therefore it is searched in the default policy table at first, then be searched in SDL Core to obtain the default appHMIType priority list. After that, the policy table has been created, so it is searched in the order, policy table->default policy table->SDL Core.
+
+However, since the preferred AppHMIType priority may vary for each OEM, the OEMs can create the priority list information of AppHMIType and store it in the policy table. By changing the priority list of AppHMIType created by each OEM in the policy table, they can display the SDL app icon according to each OEM's intended order.
+
+<b>2. Clarification of icon display sequence</b>
+Currently, there is no sequence provided that demonstrates until the icon is displayed. Figure 1 shows the sequence in which the process of this proposal added to the existing sequence from the program code of SDL Core. The black text indicates the existing order described in the program code. The red text indicates the added process by this proposal.
+
+<b>Figure 1.</b> SDL App Icon display sequence
+![Figure1_SDL_App_Icon_display_sequence.PNG](../assets/proposals/NNNN-Clarification-of-SDL-App-Icon-display-sequence-and-display-order/Figure1_SDL_App_Icon_display_sequence.PNG)
+
+The contents suggested by this proposal are as follows:
+- Add SortList() as a process to sort according to the display order of SDL App Icon.
+- Perform the processing of SortList() after processing the normal makelist(Applications_, Applications)
+- Add init() as a process to initialize the AppList used in SortList() every launch, and keep the priority list information of AppHMIType acquired from PolicyTable, etc.
+
+## Potential downsides
+
+There is no potential downside since it is a modification to the current process of the SDL app icon display order.
+
+## Impact on existing code
+
+This proposal has no impact on the existing code because there are no changes in RPC and parameters.
```

Comment:
Please advise on the impact to SDL Core and the Policy Server, as these were noted as `Impacted Platforms` on line 6.

Reply:
Hi. @theresalech -san,
Thank you for your review. I have questions about your advise. I think Impacted Platforms on line 6 means the platforms which modification occurs. On the other hands, I think Impact on existing code means whether any existing code is affected by adding the modification. 
Therefore, I think this proposal has no impact on existing code, because this it is only adding the process and table.
My understanding is correct? If incorrect, please teach me the details. Thank you.

Reply:
Here is the requested content for this section from the proposal template: `Describe the impact that this change will have on existing code. Will some SDL integrations stop compiling due to this change? Will applications still compile but produce different behavior than they used to? Is it possible to migrate existing SDL code to use a new feature or API automatically?`

It's recommended to include as much detail on the impact this proposal will have as possible.  At the very least, please note if this will require a minor version change to SDL Core and/or the Policy Server.

Reply:
Hi, @theresalech -san,
Thank you for your description. 
I will modify the PR file and make a commit it. Thank you.